src/widget/sidebar/extra.c

```diff
@@ -569,8 +637,8 @@ int sidebar_extra_handle_mouse(const mouse *m)
         return 1;
     }
     if ((data.info_to_display & SIDEBAR_EXTRA_DISPLAY_REQUESTS) &&
-        generic_buttons_handle_mouse(m, data.x_offset, data.request_buttons_y_offset,
-        buttons_emperor_requests, data.visible_requests, &data.focused_request_button_id)) {
+            generic_buttons_handle_mouse(m, data.x_offset, data.request_buttons_y_offset,
```

Comment:
Here too. Instead of having the `y` button position calculated statically, you'd need to change it dynamically, depending on which button was the `View all` button. If you don't want to do it then keep the button as-is and I'll fix it later.

Reply:
yes you can modify after